pkg/agents/team2/reward.go

```diff
@@ -0,0 +1,29 @@
+package team2
+
+func calcReward(hp int, hpInc int) float32 {
+	//TODO: reward should be redone later according to new healthInfo
+	ret := float32(0.0)
+
+	//we encourage agent to survive
+	if hp > 20 {
+		ret += 1.0
+	} else {
+		ret -= 0.5
+	}
```

Comment:
I'm assuming this was based on the assumption that hp decays by 20 each day.

Reply:
Yes, the logic has to be rewritten for next version (together with message passing).